hpedockerplugin/hpe/hpe3par_opts.py

```diff
@@ -11,6 +11,9 @@
                default='',
                help="3PAR username with the 'edit' role",
                deprecated_name='hp3par_username'),
+    cfg.StrOpt('mount_prefix',
```

Comment:
Do we need to support this per backend or per host? If per host then we can remove it from here.

Reply:
It's per host (across backends)